content/docs/studio/user-guide/views/explore-experiments.md

```diff
@@ -0,0 +1,116 @@
+# Explore ML Experiments
+
+When you create a view by connecting to a Git repository, the view is added to
+your DVC Studio dashboard. You can dive deep into the views shown in your DVC
```

Comment:
Did anything change from https://dvc.org/doc/studio/user-guide/explore-experiments ? Git didn't catch the move, instead it shows content/docs/studio/user-guide/explore-experiments.md deleted and this one crated.

Reply:
Ok yeah I see it's different from https://dvc-org-hide-commits-8bc2l1z20.herokuapp.com/doc/studio/user-guide/views/explore-experiments . A summary of changes would be really useful for the review but I'll compare the deployed versions if not. Thanks

Reply:
@jorgeorpinel Here are the changes in that file:

- Added a table of contents (3 items) in the `Components of a view` section
- Updated the gif for the `Display preferences` section. But the file was updated in the `static` repo, so there are no code changes here.
- Changed the bullet points within `Display preferences` into sub-headings (####). Eg, `Filters`, `Columns`, etc.
- Added sub-section `Hide commits` within `Display preferences`.

Reply:
Thanks, will review this tomorrow (wrapping up for today!)